freightliner_cascadia_2012_dot_10002/drivers.launch

```diff
@@ -55,18 +55,10 @@ If not using simulated drivers they are activated if the respective mock argumen
   <arg name="avt_vimba_camera"               value="$(eval ('avt_vimba_camera' in arg('drivers').split()) and not arg('mock_camera'))"/>
 
   <!-- PACMOD Controller Driver Node -->
-  <include if="$(arg ssc_interface_wrapper)" file="$(find ssc_interface_wrapper)/launch/ssc_pm_cascadia_dot_10002_driver.launch">
+  <include if="$(arg ssc_interface_wrapper)" file="$(find ssc_interface_wrapper)/launch/ssc_pm_cascadia_dot_80550_driver.launch">
```

Comment:
Let's revert this change since truck 10002 (its license plate number) will use `ssc_pm_cascadia_dot_10002_driver.launch`

Reply:
Fixed!